src/main/java/org/hyperledger/aries/api/endorser/EndorseTransactionEvent.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/acapy-java-client
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.aries.api.endorser;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.hyperledger.aries.api.serializer.JsonObjectDeserializer;
+import org.hyperledger.aries.api.serializer.JsonObjectSerializer;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Webhook event: issuer_cred_rev
+ */
+@Data @NoArgsConstructor @AllArgsConstructor
+public class EndorseTransactionEvent {
+    private TransactionType type;
+    private String connectionId;
+    private String createdAt;
+    private Boolean endorserWriteTxn;
+    @JsonSerialize(using = JsonObjectSerializer.class)
+    @JsonDeserialize(using = JsonObjectDeserializer.class)
```

Comment:
I do not think the JsonObjectSerializer works on an array type. In the client the default conversion happens with Gson. The Jackson annotations are there because users started to use the models with Jackson. So I would either remove the annotations for now or add a Jackson/Gson serialization test and add a new serializer if they fail.

Reply:
I fixed this - https://github.com/hyperledger-labs/acapy-java-client/commit/26b19874e4f34926b7a03b9f6d509075708b9c2c